job-app/src/App.js

```diff
@@ -44,6 +47,8 @@ class App extends React.Component {
                             <Card.Title>{appl.CompanyName}</Card.Title>
                             <div>{appl.Position}</div>
                             <div>{appl.Description}</div>
+                            <Button variant="primary" className="btn-primary" 
```

Comment:
You don't need to do "classname='btn-primary'". Since this is a react-bootstrap component, that variant option is the same as className